data/migrations/20200121135135_first-cars-table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", table => {
+    table.increments();
+    table.text("make").notNullable();
+    table.text("model").notNullable();
+    table
+      .text("vin")
+      .notNullable()
+      .unique();
+    table.integer("mileage").notNullable();
+    table.enu("transmissionType", ["Manual", "Semi-Automatic", "Automatic"]);
+    table.text("status");
+  });
+};
```

Comment:
It's a good practice to use data types the reflect the need for the fields. If the text expected of a field is within the range of 0 -255 characters, the appropriate data type would be varchar which is achieved with the `string` method in knex